wiki/models/__init__.py

```diff
@@ -95,8 +95,7 @@ def reverse(*args, **kwargs):
     
 # Now we redefine reverse method
 reverse_lazy = lazy(reverse, text_type)
-urlresolvers.reverse = reverse
-urlresolvers.reverse_lazy = reverse_lazy
+reverse_lazy = reverse_lazy
```

Comment:
This doesn't maintain the original behavior.  You'll probably need something like this for the changes in this file:

```
from django import shortcuts, urls

original_django_reverse = urls.reverse

<existing definition of new reverse function>

# Now we redefine reverse method
reverse_lazy = lazy(reverse, text_type)
urls.reverse = reverse
urls.reverse_lazy = reverse_lazy

# Patch up other locations of the reverse function
shortcuts.reverse = reverse
try:
    # For Django 1.11
    from django.core import urlresolvers

    urlresolvers.reverse = reverse
    urlresolvers.reverse_lazy = reverse_lazy
except ImportError:
    pass
```

Reply:
@jmbowman please review again.